tests/vmi_multus_test.go

```diff
@@ -846,6 +848,45 @@ var _ = Describe("SRIOV", func() {
 			// it's hard to match them.
 		})
 
+		Context("VLANed SR-IOV network", func() {
+			var vlanedVMI1 *v1.VirtualMachineInstance
+			cidrVlaned1 := "192.168.0.1/24"
+			cidrVlaned2 := "192.168.0.2/24"
+			cidrFlat := "192.168.1.1/24"
```

Comment:
Since you're not configuring IPAM (routing) on the sriov network's network-attach-defs, I think the IP of the "flat" VM should be in the same subnet as the others.

Reply:
I actually do not understand what is supposed to be the setup at all.
Please clarify this in the commit message so I could understand the intention and sync with the implementation.